tests/conftest.py

```diff
@@ -105,6 +112,10 @@ def consensus_mode(request):
 def blockchain_constants(consensus_mode) -> ConsensusConstants:
     if consensus_mode == Mode.PLAIN:
         return test_constants
+    if consensus_mode == Mode.HARD_FORK_2_0:
+        return test_constants.replace(
+            HARD_FORK_HEIGHT=5, PLOT_FILTER_128_HEIGHT=10, PLOT_FILTER_64_HEIGHT=15, PLOT_FILTER_32_HEIGHT=20
```

Comment:
@AmineKhaldi there are tests that don't produce 5 blocks. We should explore lowering `HARD_FORK_HEIGHT` to 2